core/base/base-object.php

```diff
@@ -132,4 +134,25 @@ private function ensure_settings() {
 			$this->settings = $this->get_init_settings();
 		}
 	}
+
+	/**
+	 * Has Own Method
+	 *
+	 * Used for check whether the method passed as a parameter was declared in the current instance or inherited.
+	 *
+	 * @since 3.1.0
+	 *
+	 * @param string $method_name
+	 *
+	 * @throws \ReflectionException
+	 * @return bool True if the method was declared by the current instance, False if it was inherited.
+	 */
+	public function has_own_method( $method_name ) {
+		$reflection_method = new ReflectionMethod( $this, $method_name );
```

Comment:
Remove the `use` from line 5.
```suggestion
		$reflection_method = new \ReflectionMethod( $this, $method_name );
```

Reply:
Done